sod-server/src/commands/base/PlaceRoadCommand.ts

```diff
@@ -14,3 +14,18 @@ export class PlaceRoadCommand extends Command<MyRoom, Payload> {
     }
   }
 }
+
+export class PlaceRoadWithPayCommand extends Command<MyRoom, Payload> {
+  execute(payload: Payload) {
+    const player = this.room.state.players.get(payload.playerId)
+    const availableRoad = player.roads.find((x) => !x.edge)
+    if (availableRoad) {
```

Comment:
fyri at sleppa undan at hava ov stórar `if` blokkar, kanst tú siga hetta ístaðin:
```ts
if(!availableRoad){
   return;
}
```